module-services/service-desktop/endpoints/include/endpoints/EndpointType.hpp

```diff
@@ -25,7 +24,8 @@ namespace sdesktop::endpoints
         bluetooth,
         usbSecurity,
         outbox,
-        reboot
+        reboot,
+        timeSync
     };
```

Comment:
Consider using a capital letter for enums ;)

Reply:
Done!